dali/kernels/test/test_utils_test.cc

```diff
@@ -48,8 +48,8 @@ TEST(TestUtil, RandomFillList) {
   const int W2 = 40;
   const int H2 = 33;
   const int N = D1*W1*H1 + D2*W2*H2;
-  float memory[N];
-  TensorListView<StorageCPU, float, 3> view(memory, { { D1, W1, H1 }, { D2, W2, H2 } });
+  std::array<float, N> memory;
```

Comment:
It's not safer - just longer...

Reply:
It is not about safety but the allocation of huge data on the stack in the above case. Change it here too for consistency.

Reply:
But that still allocates on stack. If it's to big, change it to vector or something.

Reply:
Done